scout/server/blueprints/variant/templates/variant/clinvar.html

```diff
@@ -238,7 +231,7 @@ <h5>
                     {% if pinned.category == 'snv'%}
                       <tr>
                         <td>var. identifier</td>
-                        <td><input type="text" id="Variation identifiers_{{pinned._id}}" name="variations_ids@{{pinned._id}}" value="{{ pinned.dbsnp_id.split(':')[0] if pinned.dbsnp_id else ''}}" size=14 pattern="rs[0-9]+"></td>
+                        <td><input type="text" id="Variation identifiers_{{pinned._id}}" name="variations_ids@{{pinned._id}}" value="{{ pinned.dbsnp_id.split(':')[0] if pinned.dbsnp_id else ''}}" size=14 pattern="rs[0-9]+" ></td>
```

Comment:
Not sure we need that space, but I take it you rewrote that line a few times. 😬

Reply:
I'm not really sure how it ended up there, but shouldn't affect anything